content/projects/morse-code/_index.md

```diff
@@ -57,4 +58,19 @@ For a problem like this one, unit tests are more appropriate.
 
 This is optional.
 
-Add unit tests to your code. Leave the assertions in there just so that we can see that you know how to hit the requirements we laid out. But if you have some passing tests as well that would be sweet.
\ No newline at end of file
+Add unit tests to your code. Leave the assertions in there just so that we can see that you know how to hit the requirements we laid out. But if you have some passing tests as well that would be sweet.
+
+## Instructions for Reviewers
+
+- Ensure that the lettersToMorseCode function can convert numbers, special characters and alphabets to morseCode and the morseCodeToLetters function can convert morse code to numbers, alphabets and special characters.
```

Comment:
rather say ...numbers, special characters and letters of the alphabet.
An alphabet is something that contains a lot of different letters. 

Also look over capitalisation.

And inline code can be surrounded by single backtickls to make it appear as code. Eg `lettersToMorseCode`